src/ontodia/data/sparql/sparqlDataProvider.ts

```diff
@@ -132,11 +135,24 @@ export class SparqlDataProvider implements DataProvider {
     }
 
     elementInfo(params: { elementIds: string[]; }): Promise<Dictionary<ElementModel>> {
-        const ids = params.elementIds.map(escapeIri).map(id => ` (${id})`).join(' ');
+        const blankIds: string[] = [];
+
+        const elementIds = params.elementIds.filter(id => !BlankNodes.isEncodedBlank(id));
+        const additionalData: SparqlResponse<ElementBinding> = (
```

Comment:
You could make this statement more compact (and use `undefined` instead of `null`):
```
const additionalData = this.options.acceptBlankNodes
    ? BlankNodes.elementInfo(params.elementIds) : undefined;
```